src/panels/energy/ha-panel-energy.ts

```diff
@@ -81,6 +86,19 @@ class PanelEnergy extends LitElement {
                     collectionKey="energy_dashboard"
                   ></hui-energy-period-selector>
                 `}
+            <ha-button-menu
+              corner="BOTTOM_START"
+              @action=${this._toggleCompare}
+            >
+              <ha-icon-button
+                .label=${this.hass.localize("panel.menu")}
+                .path=${mdiDotsVertical}
+                slot="trigger"
+              ></ha-icon-button>
+              <ha-check-list-item left .selected=${this._compare}>
```

Comment:
Let's show this as a "comparing mode enabled" bar in the UI at the top. Probably ha-alert?